content/post/2025-11-15-anybatross-2025fukuoka.markdown

```
@@ -0,0 +1,412 @@
+コードゴルフとは、与えられた仕様に対してできる限り短いソースコードで解答するというゲームです。
+なんだか面白そうなのでチャレンジしてみました。
+
+## Hole 1
+
+> アルファベットのAやBにあるような、文字の中にある閉じた空間のことをカウンターといいます。
+>
+> 0〜9までの10種と、アルファベット大文字のA〜Zの26種、合計36種の文字やその他の記号を利用した文字列が渡されるので、カウンターの数を数えてください。アルファベット小文字は来ません。その他の記号のカウンターは数えなくてよいです。
+>
+> 1行ずつ数えて、その行までの累積個数と、その行での出現個数を出力してください。
+>
+> - https://perlbatross.kayac.com/contest/2025fukuoka/challenge/1/sample
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)